fulu/_base_aug.py

```diff
@@ -1,21 +1,63 @@
 from abc import ABC, abstractmethod
-
 import numpy as np
+import matplotlib
+import matplotlib.colors as mcolors
+import matplotlib.pyplot as plt
+import pandas as pd
 
 
 def add_log_lam(passband, passband2lam):
+    """
+    """
+    
     log_lam = np.array([passband2lam[i] for i in passband])
     return log_lam
 
-
 def create_aug_data(t_min, t_max, passband_ids, n_obs=1000):
+    """
+    """
+    
     t = []
     passband = []
     for band in passband_ids:
         t += list(np.linspace(t_min, t_max, n_obs))
         passband += [band] * n_obs
     return np.array(t), np.array(passband)
 
+def create_colors_dict(passband2lam):
+    """
+    """
+    
+    colors = {}
+    tableau_colors = mcolors.TABLEAU_COLORS.keys()
+    for key, tone in zip(passband2lam.keys(), tableau_colors):
+        colors.update([[key, tone]])
```

Comment:
Please do not use `update` to add a single element, use `colors[key] = tone` instead. Also it can be re-written as a one-line dict comprehension, for example:
```python
colors = {band: color for band, color in zip(passband2lam, mcolors.TABLEAU_COLORS)}
```

Reply:
self.colors = {key: tone for key, tone in zip(passband2lam.keys(), mcolors.TABLEAU_COLORS.keys())}

in commit "added changes on your comments"